offload/liboffload/API/Common.td

```diff
@@ -148,6 +148,16 @@ def : Struct {
   ];
 }
 
+def : Struct {
+  let name = "ol_range_t";
```

Comment:
To me 'range' is a slightly overloaded term and doesn't necessarily imply a `size_t[3]`, `ol_dimensions_t` might be a better name. Just my opinion though, don't want to bikeshed on names too much.

Reply:
Honestly, I was unsure what to call it, or even if it was worth having a dedicated struct for it at all. I think `ol_dimensions_t` is probably better than range though, thanks.